### PR TITLE
fix(cli): run on iOS device on Xcode 14

### PR DESCRIPTION
**Pull request by lucasfernog**
### What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Docs
- [ ] New Binding issue #___
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

### Does this PR introduce a breaking change?

- [ ] Yes, and the changes were approved in issue #___
- [ ] No

### Checklist
- [ ] When resolving issues, they are referenced in the PR's title (e.g `fix: remove a typo, closes #___, #___`)
- [ ] A change file is added if any packages will require a version bump due to this PR per [the instructions in the readme](https://github.com/tauri-apps/tauri/blob/dev/.changes/readme.md).
- [ ] I have added a convincing reason for adding this feature, if necessary

### CODE REVIEW

File: tooling/cli/templates/mobile/ios/project.yml [hunk 1]
```diff
@@ -31,6 +31,7 @@ targets:
     sources:
       - path: Sources
       - path: Assets.xcassets
+      - path: Externals
```
Comment (wusyong) [inline tooling/cli/templates/mobile/ios/project.yml:34]: Can we just use Sources directory above?

File: tooling/cli/templates/mobile/ios/project.yml [hunk 2]
```diff
@@ -71,9 +72,9 @@ targets:
         ENABLE_BITCODE: false
         ARCHS: [{{join ios-valid-archs}}]
         VALID_ARCHS: {{~#each ios-valid-archs}} {{this}} {{/each}}
-        LIBRARY_SEARCH_PATHS[arch=x86_64]: $(inherited) "{{prefix-path "target/x86_64-apple-ios/$(CONFIGURATION)"}}"
-        LIBRARY_SEARCH_PATHS[arch=arm64]: $(inherited) "{{prefix-path "target/aarch64-apple-ios/$(CONFIGURATION)"}}"
-        LIBRARY_SEARCH_PATHS[arch=arm64-sim]: $(inherited) "{{prefix-path "target/aarch64-apple-ios-sim/$(CONFIGURATION)"}}"
+        LIBRARY_SEARCH_PATHS[arch=x86_64]: $(inherited) $(PROJECT_DIR)/Externals/x86_64-apple-ios/$(CONFIGURATION)
+        LIBRARY_SEARCH_PATHS[arch=arm64]: $(inherited) $(PROJECT_DIR)/Externals/aarch64-apple-ios/$(CONFIGURATION)
```
Comment (wusyong) [inline tooling/cli/templates/mobile/ios/project.yml:76]: Do you think file should be copied on simulators too?

Comment (lucasfernog) [reply tooling/cli/templates/mobile/ios/project.yml:76]: For now I'm copying them for all targets. Note that this is hopefully a temporary workaround until Apple fixes it (though I couldn't find a reported issue for this).

File: tooling/cli/templates/mobile/ios/project.yml (hunk 1 above)
Comment (lucasfernog) [reply tooling/cli/templates/mobile/ios/project.yml:34]: Since this is hopefuly temporary, I'll keep it in a separate folder.

**Pull request merged**